application/src/main/java/org/togetherjava/tjbot/commands/moderation/ReportCommand.java

```diff
@@ -154,43 +155,51 @@ private boolean isAuthorOnCooldown(long userId) {
             .isPresent();
     }
 
-    private MessageCreateAction createModMessage(String modalMessage, long userID,
-            String reportedMessage, String reportedMessageUrl, TextChannel modMailAuditLog) {
-        MessageEmbed embed = new EmbedBuilder().setTitle("Report")
-            .setDescription(("""
+    private MessageCreateAction createModMessage(String reportReason, User author,
+            String reportedMessage, String reportedMessageUrl, String reportedMessageTimestamp,
+            TextChannel modMailAuditLog) {
+        MessageEmbed reportedMessageEmbed = new EmbedBuilder().setTitle("Report")
+            .setDescription("""
                     Reported Message: **%s**
-                    [Reported Message URL](%s)
-                    Reason: **%s**""").formatted(reportedMessage, reportedMessageUrl, modalMessage))
-            .setAuthor("Author ID: %d".formatted(userID))
+                    [Reported Message URL](%s)""".formatted(reportedMessage, reportedMessageUrl))
```

Comment:
id like to have the url as button or in the footer or something.
```java
.setDescription(reportedMessage)
.setFooter(reportedMessageUrl) // or add a button
```
it will look much more natural that way and less "constructed"

Reply:
It doesn't work in the footer, il make a button